### PR TITLE
Update version string and add warning on import.

### PR DESCRIPTION
**Pull request by twiecki**

### CODE REVIEW

File: pymc3/__init__.py [hunk 1]
```diff
@@ -27,6 +27,10 @@
         handler = logging.StreamHandler()
         _log.addHandler(handler)
 
+_log.info(
+    "You are running the development version of PyMC3 which currently still lacks key features."
```
Comment (ricardoV94) [inline pymc3/__init__.py]: Perhaps add something along the lines of:

> The latest stable release is version `3.11.2`.

**Comment by brandonwillard** [comment]
Side note: have we considered setting up `versioneer`?

https://github.com/pymc-devs/pymc3/issues/4727

**Review by brandonwillard — APPROVED** [review]
These changes look fine, but I don't know what the failures are about, yet.

**Review by brandonwillard — CHANGES_REQUESTED** [review]
Ah, looks like [the error](https://github.com/pymc-devs/pymc3/pull/4726/checks?check_run_id=2683532131#step:7:311) is due to the version string parsing in Arviz.

I created an issue: https://github.com/arviz-devs/arviz/issues/1708.

File: pymc3/__init__.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 #   limitations under the License.
 
 # pylint: disable=wildcard-import
-__version__ = "3.11.1"
+__version__ = "4.0.dev0"
```
Comment (twiecki) [inline pymc3/__init__.py]:
```suggestion
__version__ = "4.0"
```

**Pull request merged**